2025-07-11T09:30:16+0000 - faraday.server.app - INFO {MainThread} [pid:64] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-11T09:30:16+0000 - faraday.server.app - INFO {MainThread} [pid:64] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-11T09:30:22+0000 - faraday.server.app - INFO {MainThread} [pid:68] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-11T09:30:22+0000 - faraday.server.app - INFO {MainThread} [pid:68] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-11T09:31:34+0000 - audit - WARNING {Dummy-5} [pid:64] [app.py:707 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 09:31:34.274396] - Reason: [Invalid Password]
2025-07-11T09:31:34+0000 - faraday.server.app - WARNING {Dummy-5} [pid:64] [app.py:709 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 09:31:34.274396] - Reason: [Invalid Password]
2025-07-11T09:33:46+0000 - audit - WARNING {Dummy-9} [pid:64] [app.py:707 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 09:33:46.079156] - Reason: [Invalid Password]
2025-07-11T09:33:46+0000 - faraday.server.app - WARNING {Dummy-9} [pid:64] [app.py:709 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 09:33:46.079156] - Reason: [Invalid Password]
2025-07-11T09:33:55+0000 - audit - WARNING {Dummy-9} [pid:64] [app.py:707 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 09:33:54.519215] - Reason: [Invalid Password]
2025-07-11T09:33:55+0000 - faraday.server.app - WARNING {Dummy-9} [pid:64] [app.py:709 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 09:33:54.519215] - Reason: [Invalid Password]
2025-07-11T09:34:56+0000 - audit - WARNING {Dummy-15} [pid:64] [app.py:707 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 09:34:56.136189] - Reason: [Invalid Password]
2025-07-11T09:34:56+0000 - faraday.server.app - WARNING {Dummy-15} [pid:64] [app.py:709 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 09:34:56.136189] - Reason: [Invalid Password]
2025-07-11T09:35:04+0000 - audit - WARNING {Dummy-15} [pid:64] [app.py:690 - validate()]  Invalid Login - User [Faraday] from IP [172.23.0.1] at [2025-07-11 09:35:04.560783] - Reason: [Invalid Username]
2025-07-11T09:35:04+0000 - faraday.server.app - WARNING {Dummy-15} [pid:64] [app.py:692 - validate()]  Invalid Login - User [Faraday] from IP [172.23.0.1] at [2025-07-11 09:35:04.560783] - Reason: [Invalid Username]
2025-07-11T09:35:52+0000 - audit - WARNING {Dummy-16} [pid:64] [app.py:682 - validate()]  Invalid Login - User [None] from IP [172.23.0.1] at [2025-07-11 09:35:52.461979]
2025-07-11T09:35:52+0000 - faraday.server.app - WARNING {Dummy-16} [pid:64] [app.py:683 - validate()]  Invalid Login - User [None] from IP [172.23.0.1] at [2025-07-11 09:35:52.461979]
2025-07-11T09:42:13+0000 - audit - WARNING {Dummy-17} [pid:64] [app.py:682 - validate()]  Invalid Login - User [None] from IP [172.23.0.1] at [2025-07-11 09:42:13.343248]
2025-07-11T09:42:13+0000 - faraday.server.app - WARNING {Dummy-17} [pid:64] [app.py:683 - validate()]  Invalid Login - User [None] from IP [172.23.0.1] at [2025-07-11 09:42:13.343248]
2025-07-11T09:43:45+0000 - audit - WARNING {Dummy-18} [pid:64] [app.py:707 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 09:43:45.131831] - Reason: [Invalid Password]
2025-07-11T09:43:45+0000 - faraday.server.app - WARNING {Dummy-18} [pid:64] [app.py:709 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 09:43:45.131831] - Reason: [Invalid Password]
2025-07-11T09:47:01+0000 - audit - WARNING {Dummy-20} [pid:64] [app.py:690 - validate()]  Invalid Login - User [admin] from IP [172.23.0.1] at [2025-07-11 09:47:01.551211] - Reason: [Invalid Username]
2025-07-11T09:47:01+0000 - faraday.server.app - WARNING {Dummy-20} [pid:64] [app.py:692 - validate()]  Invalid Login - User [admin] from IP [172.23.0.1] at [2025-07-11 09:47:01.551211] - Reason: [Invalid Username]
2025-07-11T11:19:59+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-11T11:19:59+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-11T11:20:03+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-11T11:20:03+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-11T11:21:47+0000 - audit - WARNING {Dummy-7} [pid:23] [app.py:707 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 11:21:47.192938] - Reason: [Invalid Password]
2025-07-11T11:21:47+0000 - faraday.server.app - WARNING {Dummy-7} [pid:23] [app.py:709 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 11:21:47.192938] - Reason: [Invalid Password]
2025-07-11T11:21:54+0000 - audit - WARNING {Dummy-7} [pid:23] [app.py:682 - validate()]  Invalid Login - User [] from IP [172.23.0.1] at [2025-07-11 11:21:54.387422]
2025-07-11T11:21:54+0000 - faraday.server.app - WARNING {Dummy-7} [pid:23] [app.py:683 - validate()]  Invalid Login - User [] from IP [172.23.0.1] at [2025-07-11 11:21:54.387422]
2025-07-11T11:24:12+0000 - audit - WARNING {Dummy-7} [pid:23] [app.py:707 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 11:24:12.303566] - Reason: [Invalid Password]
2025-07-11T11:24:12+0000 - faraday.server.app - WARNING {Dummy-7} [pid:23] [app.py:709 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 11:24:12.303566] - Reason: [Invalid Password]
2025-07-11T11:29:11+0000 - audit - WARNING {Dummy-7} [pid:23] [app.py:707 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 11:29:11.122811] - Reason: [Invalid Password]
2025-07-11T11:29:11+0000 - faraday.server.app - WARNING {Dummy-7} [pid:23] [app.py:709 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 11:29:11.122811] - Reason: [Invalid Password]
2025-07-11T11:30:14+0000 - audit - WARNING {Dummy-7} [pid:23] [app.py:707 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 11:30:13.675030] - Reason: [Invalid Password]
2025-07-11T11:30:14+0000 - faraday.server.app - WARNING {Dummy-7} [pid:23] [app.py:709 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 11:30:13.675030] - Reason: [Invalid Password]
2025-07-11T11:34:52+0000 - audit - WARNING {Dummy-7} [pid:23] [app.py:707 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 11:34:52.386635] - Reason: [Invalid Password]
2025-07-11T11:34:52+0000 - faraday.server.app - WARNING {Dummy-7} [pid:23] [app.py:709 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 11:34:52.386635] - Reason: [Invalid Password]
2025-07-11T11:35:00+0000 - audit - WARNING {Dummy-7} [pid:23] [app.py:707 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 11:34:59.766655] - Reason: [Invalid Password]
2025-07-11T11:35:00+0000 - faraday.server.app - WARNING {Dummy-7} [pid:23] [app.py:709 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 11:34:59.766655] - Reason: [Invalid Password]
2025-07-11T11:35:01+0000 - audit - WARNING {Dummy-7} [pid:23] [app.py:707 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 11:35:01.610351] - Reason: [Invalid Password]
2025-07-11T11:35:01+0000 - faraday.server.app - WARNING {Dummy-7} [pid:23] [app.py:709 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-11 11:35:01.610351] - Reason: [Invalid Password]
2025-07-11T11:38:52+0000 - faraday.server.app - WARNING {Dummy-9} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-11T12:42:07+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-11T12:42:07+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-11T12:42:11+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-11T12:42:11+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-11T12:48:37+0000 - audit - INFO {Dummy-6} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-11 12:48:37.947022]
2025-07-11T12:48:37+0000 - faraday.server.app - INFO {Dummy-6} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-11 12:48:37.947022]
2025-07-13T15:51:40+0000 - audit - INFO {Dummy-16} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-13 15:51:40.615315]
2025-07-13T15:51:40+0000 - faraday.server.app - INFO {Dummy-16} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-13 15:51:40.615315]
2025-07-14T16:16:43+0000 - audit - INFO {Dummy-47} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-14 16:16:43.850677]
2025-07-14T16:16:43+0000 - faraday.server.app - INFO {Dummy-47} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-14 16:16:43.850677]
2025-07-16T14:51:32+0000 - audit - INFO {Dummy-77} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-16 14:51:32.161918]
2025-07-16T14:51:32+0000 - faraday.server.app - INFO {Dummy-77} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-16 14:51:32.161918]
2025-07-16T18:15:51+0000 - faraday.server.app - WARNING {Dummy-262} [pid:23] [app.py:228 - load_user_from_request()]  Authorization header does not have type
2025-07-16T18:17:40+0000 - faraday.server.app - WARNING {Dummy-290} [pid:23] [app.py:228 - load_user_from_request()]  Authorization header does not have type
2025-07-16T18:18:30+0000 - faraday.server.app - WARNING {Dummy-302} [pid:23] [app.py:228 - load_user_from_request()]  Authorization header does not have type
2025-07-16T18:23:13+0000 - faraday.server.app - WARNING {Dummy-369} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-16T18:24:13+0000 - faraday.server.app - WARNING {Dummy-392} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-16T18:25:48+0000 - faraday.server.app - WARNING {Dummy-420} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-16T18:26:10+0000 - faraday.server.app - WARNING {Dummy-429} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-17T14:21:46+0000 - faraday.server.app - WARNING {Dummy-737} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-17T14:23:26+0000 - faraday.server.app - WARNING {Dummy-778} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-17T14:23:36+0000 - faraday.server.app - WARNING {Dummy-784} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-17T15:32:20+0000 - audit - INFO {Dummy-1186} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-17 15:32:20.704439]
2025-07-17T15:32:20+0000 - faraday.server.app - INFO {Dummy-1186} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-17 15:32:20.704439]
2025-07-18T15:24:11+0000 - audit - WARNING {Dummy-1914} [pid:23] [app.py:682 - validate()]  Invalid Login - User [None] from IP [172.23.0.1] at [2025-07-18 15:24:11.171229]
2025-07-18T15:24:11+0000 - faraday.server.app - WARNING {Dummy-1914} [pid:23] [app.py:683 - validate()]  Invalid Login - User [None] from IP [172.23.0.1] at [2025-07-18 15:24:11.171229]
2025-07-18T16:38:33+0000 - faraday.server.app - WARNING {Dummy-2115} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-18T16:39:29+0000 - audit - WARNING {Dummy-2115} [pid:23] [app.py:682 - validate()]  Invalid Login - User [None] from IP [172.23.0.1] at [2025-07-18 16:39:29.421154]
2025-07-18T16:39:29+0000 - faraday.server.app - WARNING {Dummy-2115} [pid:23] [app.py:683 - validate()]  Invalid Login - User [None] from IP [172.23.0.1] at [2025-07-18 16:39:29.421154]
2025-07-18T16:50:13+0000 - audit - WARNING {Dummy-2203} [pid:23] [app.py:682 - validate()]  Invalid Login - User [None] from IP [172.23.0.1] at [2025-07-18 16:50:13.048176]
2025-07-18T16:50:13+0000 - faraday.server.app - WARNING {Dummy-2203} [pid:23] [app.py:683 - validate()]  Invalid Login - User [None] from IP [172.23.0.1] at [2025-07-18 16:50:13.048176]
2025-07-18T16:50:19+0000 - audit - WARNING {Dummy-2203} [pid:23] [app.py:682 - validate()]  Invalid Login - User [None] from IP [172.23.0.1] at [2025-07-18 16:50:19.947510]
2025-07-18T16:50:19+0000 - faraday.server.app - WARNING {Dummy-2203} [pid:23] [app.py:683 - validate()]  Invalid Login - User [None] from IP [172.23.0.1] at [2025-07-18 16:50:19.947510]
2025-07-18T16:50:26+0000 - audit - WARNING {Dummy-2203} [pid:23] [app.py:682 - validate()]  Invalid Login - User [None] from IP [172.23.0.1] at [2025-07-18 16:50:26.836387]
2025-07-18T16:50:26+0000 - faraday.server.app - WARNING {Dummy-2203} [pid:23] [app.py:683 - validate()]  Invalid Login - User [None] from IP [172.23.0.1] at [2025-07-18 16:50:26.836387]
2025-07-18T16:54:54+0000 - audit - INFO {Dummy-2214} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-18 16:54:54.727193]
2025-07-18T16:54:54+0000 - faraday.server.app - INFO {Dummy-2214} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-18 16:54:54.727193]
2025-07-18T16:55:03+0000 - audit - INFO {Dummy-2214} [pid:23] [app.py:345 - expire_session()]  User [faraday] logged out from IP [172.23.0.1] at [2025-07-18 16:55:03.457058]
2025-07-18T16:55:03+0000 - faraday.server.app - INFO {Dummy-2214} [pid:23] [app.py:346 - expire_session()]  User [faraday] logged out from IP [172.23.0.1] at [2025-07-18 16:55:03.457058]
2025-07-18T16:55:26+0000 - audit - WARNING {Dummy-2214} [pid:23] [app.py:707 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-18 16:55:26.015066] - Reason: [Invalid Password]
2025-07-18T16:55:26+0000 - faraday.server.app - WARNING {Dummy-2214} [pid:23] [app.py:709 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-18 16:55:26.015066] - Reason: [Invalid Password]
2025-07-18T16:55:35+0000 - audit - INFO {Dummy-2214} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-18 16:55:35.597694]
2025-07-18T16:55:35+0000 - faraday.server.app - INFO {Dummy-2214} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-18 16:55:35.597694]
2025-07-18T16:56:40+0000 - audit - INFO {Dummy-2214} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-18 16:56:40.694222]
2025-07-18T16:56:40+0000 - faraday.server.app - INFO {Dummy-2214} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-18 16:56:40.694222]
2025-07-18T16:57:01+0000 - audit - INFO {Dummy-2214} [pid:23] [app.py:345 - expire_session()]  User [faraday] logged out from IP [172.23.0.1] at [2025-07-18 16:57:01.786131]
2025-07-18T16:57:01+0000 - faraday.server.app - INFO {Dummy-2214} [pid:23] [app.py:346 - expire_session()]  User [faraday] logged out from IP [172.23.0.1] at [2025-07-18 16:57:01.786131]
2025-07-18T16:57:05+0000 - audit - WARNING {Dummy-2214} [pid:23] [app.py:707 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-18 16:57:04.912195] - Reason: [Invalid Password]
2025-07-18T16:57:05+0000 - faraday.server.app - WARNING {Dummy-2214} [pid:23] [app.py:709 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-18 16:57:04.912195] - Reason: [Invalid Password]
2025-07-18T16:57:19+0000 - audit - INFO {Dummy-2214} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-18 16:57:19.482625]
2025-07-18T16:57:19+0000 - faraday.server.app - INFO {Dummy-2214} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-18 16:57:19.482625]
2025-07-20T16:04:34+0000 - audit - WARNING {Dummy-2580} [pid:23] [app.py:707 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-20 16:04:33.296402] - Reason: [Invalid Password]
2025-07-20T16:04:34+0000 - faraday.server.app - WARNING {Dummy-2580} [pid:23] [app.py:709 - validate()]  Invalid Login - User [faraday] from IP [172.23.0.1] at [2025-07-20 16:04:33.296402] - Reason: [Invalid Password]
2025-07-20T16:04:49+0000 - audit - INFO {Dummy-2581} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-20 16:04:49.596630]
2025-07-20T16:04:49+0000 - faraday.server.app - INFO {Dummy-2581} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-20 16:04:49.596630]
2025-07-22T14:49:06+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-22T14:49:06+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-22T14:49:11+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-22T14:49:11+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-22T14:51:43+0000 - faraday.server.app - WARNING {Dummy-12} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-22T15:02:15+0000 - audit - INFO {Dummy-40} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-22 15:02:15.663183]
2025-07-22T15:02:15+0000 - faraday.server.app - INFO {Dummy-40} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-22 15:02:15.663183]
2025-07-22T16:00:09+0000 - faraday.server.app - WARNING {Dummy-130} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-22T16:10:59+0000 - faraday.server.app - WARNING {Dummy-151} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-22T16:55:03+0000 - faraday.server.app - WARNING {Dummy-289} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-22T17:01:08+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-22T17:01:08+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-22T17:01:13+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-22T17:01:13+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-22T17:34:17+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-22T17:34:17+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-22T17:34:22+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-22T17:34:22+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-22T18:11:31+0000 - faraday.server.app - INFO {MainThread} [pid:24] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-22T18:11:31+0000 - faraday.server.app - INFO {MainThread} [pid:24] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-22T18:11:42+0000 - faraday.server.app - INFO {MainThread} [pid:28] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-22T18:11:42+0000 - faraday.server.app - INFO {MainThread} [pid:28] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-22T18:11:53+0000 - audit - INFO {Dummy-18} [pid:24] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-22 18:11:53.550843]
2025-07-22T18:11:53+0000 - faraday.server.app - INFO {Dummy-18} [pid:24] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-22 18:11:53.550843]
2025-07-22T18:40:07+0000 - faraday.server.app - WARNING {Dummy-138} [pid:24] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-24T14:39:10+0000 - audit - INFO {Dummy-643} [pid:24] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-24 14:39:10.767195]
2025-07-24T14:39:10+0000 - faraday.server.app - INFO {Dummy-643} [pid:24] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-24 14:39:10.767195]
2025-07-24T15:53:24+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-24T15:53:24+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-24T15:53:34+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-24T15:53:34+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-24T16:39:23+0000 - audit - INFO {Dummy-191} [pid:23] [app.py:345 - expire_session()]  User [faraday] logged out from IP [172.23.0.1] at [2025-07-24 16:39:23.084352]
2025-07-24T16:39:23+0000 - faraday.server.app - INFO {Dummy-191} [pid:23] [app.py:346 - expire_session()]  User [faraday] logged out from IP [172.23.0.1] at [2025-07-24 16:39:23.084352]
2025-07-24T16:39:25+0000 - audit - INFO {Dummy-191} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-24 16:39:25.854478]
2025-07-24T16:39:25+0000 - faraday.server.app - INFO {Dummy-191} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-24 16:39:25.854478]
2025-07-24T16:39:29+0000 - audit - INFO {Dummy-191} [pid:23] [app.py:345 - expire_session()]  User [faraday] logged out from IP [172.23.0.1] at [2025-07-24 16:39:29.133786]
2025-07-24T16:39:29+0000 - faraday.server.app - INFO {Dummy-191} [pid:23] [app.py:346 - expire_session()]  User [faraday] logged out from IP [172.23.0.1] at [2025-07-24 16:39:29.133786]
2025-07-24T16:39:41+0000 - audit - INFO {Dummy-191} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-24 16:39:41.288054]
2025-07-24T16:39:41+0000 - faraday.server.app - INFO {Dummy-191} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-24 16:39:41.288054]
2025-07-25T17:45:22+0000 - audit - INFO {Dummy-2003} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-25 17:45:22.385869]
2025-07-25T17:45:22+0000 - faraday.server.app - INFO {Dummy-2003} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-25 17:45:22.385869]
2025-07-27T23:50:46+0000 - audit - INFO {Dummy-3478} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-27 23:50:46.590044]
2025-07-27T23:50:46+0000 - faraday.server.app - INFO {Dummy-3478} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-27 23:50:46.590044]
2025-07-28T01:18:10+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-28T01:18:10+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-28T01:18:15+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-28T01:18:15+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-28T01:19:15+0000 - faraday.server.app - INFO {MainThread} [pid:24] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-28T01:19:15+0000 - faraday.server.app - INFO {MainThread} [pid:24] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-28T01:19:23+0000 - faraday.server.app - INFO {MainThread} [pid:28] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-28T01:19:23+0000 - faraday.server.app - INFO {MainThread} [pid:28] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-28T01:19:30+0000 - audit - INFO {Dummy-7} [pid:24] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-28 01:19:30.064930]
2025-07-28T01:19:30+0000 - faraday.server.app - INFO {Dummy-7} [pid:24] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-28 01:19:30.064930]
2025-07-28T01:31:23+0000 - faraday.server.app - WARNING {Dummy-38} [pid:24] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-28T01:51:59+0000 - faraday.server.app - WARNING {Dummy-88} [pid:24] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-28T16:04:55+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-28T16:04:55+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-28T16:04:59+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-28T16:04:59+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-28T16:05:42+0000 - audit - INFO {Dummy-6} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-28 16:05:42.075041]
2025-07-28T16:05:42+0000 - faraday.server.app - INFO {Dummy-6} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-28 16:05:42.075041]
2025-07-28T16:55:13+0000 - faraday.server.app - WARNING {Dummy-130} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-28T17:05:48+0000 - faraday.server.app - WARNING {Dummy-144} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-28T17:53:36+0000 - faraday.server.app - WARNING {Dummy-257} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-28T18:13:37+0000 - faraday.server.app - INFO {Thread-297} [pid:23] [debouncer.py:352 - update_workspace_vulns_count()]  workspace vulns count took 0:00:00.571029
2025-07-28T18:21:22+0000 - faraday.server.app - WARNING {Dummy-311} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-28T18:32:13+0000 - faraday.server.app - WARNING {Dummy-324} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-30T15:06:48+0000 - audit - INFO {Dummy-325} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-30 15:06:48.778187]
2025-07-30T15:06:48+0000 - faraday.server.app - INFO {Dummy-325} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-30 15:06:48.778187]
2025-07-30T15:11:09+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-30T15:11:09+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-30T15:11:17+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-30T15:11:17+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-30T15:11:18+0000 - audit - INFO {Dummy-8} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-30 15:11:18.218652]
2025-07-30T15:11:18+0000 - faraday.server.app - INFO {Dummy-8} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-30 15:11:18.218652]
2025-07-30T16:28:11+0000 - faraday.server.app - WARNING {Dummy-102} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.00
2025-07-30T16:48:17+0000 - faraday.server.app - WARNING {Dummy-154} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-30T16:55:14+0000 - faraday.server.app - WARNING {Dummy-167} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-30T17:03:50+0000 - faraday.server.app - WARNING {Dummy-188} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-30T17:46:04+0000 - faraday.server.app - WARNING {Dummy-253} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-30T18:23:09+0000 - faraday.server.app - WARNING {Dummy-318} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-07-30T18:23:21+0000 - faraday.server.app - WARNING {Dummy-317} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-07-31T00:35:46+0000 - faraday.server.app - WARNING {Dummy-387} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-31T00:38:51+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-31T00:38:51+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-31T00:38:55+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-07-31T00:38:55+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-07-31T00:39:26+0000 - audit - INFO {Dummy-6} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-31 00:39:26.415344]
2025-07-31T00:39:26+0000 - faraday.server.app - INFO {Dummy-6} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-07-31 00:39:26.415344]
2025-07-31T00:39:27+0000 - faraday.server.app - WARNING {Dummy-6} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.02
2025-07-31T01:39:33+0000 - faraday.server.app - WARNING {Dummy-96} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-31T02:02:51+0000 - faraday.server.app - WARNING {Dummy-171} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-31T02:07:25+0000 - faraday.server.app - WARNING {Dummy-198} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.02
2025-07-31T02:08:08+0000 - faraday.server.app - WARNING {Dummy-198} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.03
2025-07-31T02:20:17+0000 - faraday.server.app - WARNING {Dummy-252} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.03
2025-07-31T02:21:32+0000 - faraday.server.app - WARNING {Dummy-269} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (18) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.00
2025-07-31T02:21:34+0000 - faraday.server.app - WARNING {Dummy-273} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (20) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.00
2025-07-31T02:21:45+0000 - faraday.server.app - WARNING {Dummy-285} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (18) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.00
2025-07-31T02:21:47+0000 - faraday.server.app - WARNING {Dummy-290} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (20) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.00
2025-07-31T02:22:10+0000 - faraday.server.app - WARNING {Dummy-299} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (20) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.00
2025-07-31T02:22:40+0000 - faraday.server.app - WARNING {Dummy-324} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (20) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.00
2025-07-31T02:23:05+0000 - faraday.server.app - WARNING {Dummy-334} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (18) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.00
2025-07-31T02:23:07+0000 - faraday.server.app - WARNING {Dummy-338} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (20) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.00
2025-07-31T02:44:08+0000 - faraday.server.app - WARNING {Dummy-411} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (20) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.00
2025-07-31T08:53:54+0000 - faraday.server.app - WARNING {Dummy-514} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-31T09:02:58+0000 - faraday.server.app - WARNING {Dummy-544} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.34
2025-07-31T09:58:44+0000 - faraday.server.app - WARNING {Dummy-707} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-07-31T09:58:58+0000 - faraday.server.app - WARNING {Dummy-710} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-07-31T10:40:36+0000 - faraday.server.app - WARNING {Dummy-799} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-31T10:47:54+0000 - faraday.server.app - WARNING {Dummy-819} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-31T12:25:21+0000 - faraday.server.app - WARNING {Dummy-1027} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-31T12:38:07+0000 - faraday.server.app - WARNING {Dummy-1059} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-07-31T12:38:09+0000 - faraday.server.app - WARNING {Dummy-1061} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-07-31T13:03:16+0000 - faraday.server.app - WARNING {Dummy-1127} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-07-31T13:06:50+0000 - faraday.server.app - WARNING {Dummy-1135} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-08-05T16:51:50+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-08-05T16:51:50+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-08-05T16:51:58+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-08-05T16:51:58+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-08-05T16:52:12+0000 - audit - INFO {Dummy-10} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-08-05 16:52:12.915624]
2025-08-05T16:52:12+0000 - faraday.server.app - INFO {Dummy-10} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-08-05 16:52:12.915624]
2025-08-05T16:52:13+0000 - faraday.server.app - WARNING {Dummy-10} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-08-05T16:52:13+0000 - faraday.server.app - WARNING {Dummy-10} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-08-05T18:32:09+0000 - faraday.server.app - WARNING {Dummy-188} [pid:23] [app.py:250 - load_user_from_request()]  Invalid authorization type
2025-08-06T17:31:25+0000 - audit - INFO {Dummy-318} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-08-06 17:31:25.236620]
2025-08-06T17:31:25+0000 - faraday.server.app - INFO {Dummy-318} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-08-06 17:31:25.236620]
2025-08-06T17:31:25+0000 - faraday.server.app - WARNING {Dummy-318} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-08-11T16:09:53+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-08-11T16:09:53+0000 - faraday.server.app - INFO {MainThread} [pid:23] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-08-11T16:09:57+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:638 - register_extensions()]  Celery is enabled ...
2025-08-11T16:09:57+0000 - faraday.server.app - INFO {MainThread} [pid:27] [app.py:639 - register_extensions()]  Checking celery configuration ...
2025-08-11T16:10:01+0000 - audit - INFO {Dummy-4} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-08-11 16:10:01.770227]
2025-08-11T16:10:01+0000 - faraday.server.app - INFO {Dummy-4} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-08-11 16:10:01.770227]
2025-08-11T16:10:02+0000 - faraday.server.app - WARNING {Dummy-4} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.02
2025-08-11T16:10:02+0000 - faraday.server.app - WARNING {Dummy-4} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-08-11T17:17:42+0000 - faraday.server.app - WARNING {Dummy-116} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.06
2025-08-11T18:10:10+0000 - faraday.server.app - WARNING {Dummy-246} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.31
2025-08-11T18:35:19+0000 - faraday.server.app - WARNING {Dummy-304} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (16) in endpoint vulns_api.VulnerabilityView:index. Maximum query time: 0.04
2025-08-11T18:35:25+0000 - faraday.server.app - WARNING {Dummy-305} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (16) in endpoint vulns_api.VulnerabilityView:index. Maximum query time: 0.02
2025-08-11T18:35:31+0000 - faraday.server.app - WARNING {Dummy-306} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (16) in endpoint vulns_api.VulnerabilityView:index. Maximum query time: 0.04
2025-08-11T18:36:40+0000 - faraday.server.app - WARNING {Dummy-310} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (16) in endpoint vulns_api.VulnerabilityView:index. Maximum query time: 0.02
2025-08-13T15:23:41+0000 - audit - INFO {Dummy-345} [pid:23] [app.py:366 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-08-13 15:23:41.075930]
2025-08-13T15:23:41+0000 - faraday.server.app - INFO {Dummy-345} [pid:23] [app.py:367 - user_logged_in_successful()]  User [faraday] logged in from IP [172.23.0.1] at [2025-08-13 15:23:41.075930]
2025-08-13T15:23:41+0000 - faraday.server.app - WARNING {Dummy-345} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-08-13T15:23:47+0000 - faraday.server.app - WARNING {Dummy-345} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.02
2025-08-13T15:34:28+0000 - faraday.server.app - WARNING {Dummy-395} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.02
2025-08-13T15:34:47+0000 - faraday.server.app - WARNING {Dummy-395} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.02
2025-08-13T15:50:36+0000 - faraday.server.app - WARNING {Dummy-440} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-08-13T16:50:58+0000 - faraday.server.app - WARNING {Dummy-591} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (21) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.05
2025-08-13T17:50:21+0000 - faraday.server.app - WARNING {Dummy-1243} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-08-13T17:50:44+0000 - faraday.server.app - WARNING {Dummy-1248} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.12
2025-08-13T17:50:53+0000 - faraday.server.app - WARNING {Dummy-1244} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.16
2025-08-13T17:51:05+0000 - faraday.server.app - WARNING {Dummy-1244} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-08-13T17:51:17+0000 - faraday.server.app - WARNING {Dummy-1247} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-08-13T17:51:39+0000 - faraday.server.app - ERROR {Dummy-1275} [pid:23] [app.py:1449 - log_exception()]  Exception on /_api/v3/ws/worspacetest/vulns/1755107368857 [DELETE]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
    cursor.execute(statement, parameters)
psycopg2.errors.NumericValueOutOfRange: value "1755107368857" is out of range for type integer
LINE 3: ... (7)) AND workspace.id = 3 AND vulnerability.id = '175510736...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/site-packages/flask/app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/usr/local/lib/python3.8/site-packages/flask_classful.py", line 268, in proxy
    response = view(**request.view_args)
  File "/usr/local/lib/python3.8/site-packages/flask_classful.py", line 239, in inner
    return fn(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/faraday/server/api/base.py", line 1768, in delete
    return super().delete(object_id, workspace_name=workspace_name)
  File "/usr/local/lib/python3.8/site-packages/faraday/server/api/base.py", line 1696, in delete
    obj = self._get_object(object_id, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/faraday/server/api/base.py", line 476, in _get_object
    obj = query.filter(self._get_lookup_field() == object_id).one()
  File "/usr/local/lib/python3.8/site-packages/nplusone/core/signals.py", line 23, in wrapped
    ret = func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/orm/query.py", line 3490, in one
    ret = self.one_or_none()
  File "/usr/local/lib/python3.8/site-packages/nplusone/core/signals.py", line 23, in wrapped
    ret = func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/orm/query.py", line 3459, in one_or_none
    ret = list(self)
  File "/usr/local/lib/python3.8/site-packages/nplusone/ext/sqlalchemy.py", line 99, in query_iter
    ret, clone = itertools.tee(original_query_iter(self))
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/orm/query.py", line 3535, in __iter__
    return self._execute_and_instances(context)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/orm/query.py", line 3560, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1011, in execute
    return meth(self, multiparams, params)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/sql/elements.py", line 298, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1124, in _execute_clauseelement
    ret = self._execute_context(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1316, in _execute_context
    self._handle_dbapi_exception(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1510, in _handle_dbapi_exception
    util.raise_(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.DataError: (psycopg2.errors.NumericValueOutOfRange) value "1755107368857" is out of range for type integer
LINE 3: ... (7)) AND workspace.id = 3 AND vulnerability.id = '175510736...
                                                             ^

[SQL: SELECT vulnerability.create_date AS vulnerability_create_date, vulnerability.update_date AS vulnerability_update_date, vulnerability.data AS vulnerability_data, vulnerability.description AS vulnerability_description, vulnerability.ease_of_resolution AS vulnerability_ease_of_resolution, vulnerability.name AS vulnerability_name, vulnerability.resolution AS vulnerability_resolution, vulnerability.severity AS vulnerability_severity, vulnerability.risk AS vulnerability_risk, vulnerability.impact_accountability AS vulnerability_impact_accountability, vulnerability.impact_availability AS vulnerability_impact_availability, vulnerability.impact_confidentiality AS vulnerability_impact_confidentiality, vulnerability.impact_integrity AS vulnerability_impact_integrity, vulnerability.external_id AS vulnerability_external_id, vulnerability.custom_fields AS vulnerability_custom_fields, vulnerability.id AS vulnerability_id, vulnerability._tmp_id AS vulnerability__tmp_id, vulnerability.confirmed AS vulnerability_confirmed, vulnerability.status AS vulnerability_status, vulnerability.type AS vulnerability_type, vulnerability.issuetracker AS vulnerability_issuetracker, vulnerability.association_date AS vulnerability_association_date, vulnerability.disassociated_manually AS vulnerability_disassociated_manually, vulnerability.tool AS vulnerability_tool, vulnerability.method AS vulnerability_method, vulnerability.parameters AS vulnerability_parameters, vulnerability.parameter_name AS vulnerability_parameter_name, vulnerability.path AS vulnerability_path, vulnerability.query_string AS vulnerability_query_string, vulnerability.request AS vulnerability_request, vulnerability.response AS vulnerability_response, vulnerability.website AS vulnerability_website, vulnerability.status_code AS vulnerability_status_code, vulnerability.epss AS vulnerability_epss, vulnerability.is_main AS vulnerability_is_main, vulnerability.vulnerability_duplicate_id AS vulnerability_vulnerability_duplicate_id, vulnerability.vulnerability_template_id AS vulnerability_vulnerability_template_id, vulnerability.workspace_id AS vulnerability_workspace_id, vulnerability._cvss2_vector_string AS vulnerability__cvss2_vector_string, vulnerability.cvss2_base_score AS vulnerability_cvss2_base_score, vulnerability.cvss2_exploitability_score AS vulnerability_cvss2_exploitability_score, vulnerability.cvss2_impact_score AS vulnerability_cvss2_impact_score, vulnerability.cvss2_base_severity AS vulnerability_cvss2_base_severity, vulnerability.cvss2_temporal_score AS vulnerability_cvss2_temporal_score, vulnerability.cvss2_temporal_severity AS vulnerability_cvss2_temporal_severity, vulnerability.cvss2_environmental_score AS vulnerability_cvss2_environmental_score, vulnerability.cvss2_environmental_severity AS vulnerability_cvss2_environmental_severity, vulnerability.cvss2_access_vector AS vulnerability_cvss2_access_vector, vulnerability.cvss2_access_complexity AS vulnerability_cvss2_access_complexity, vulnerability.cvss2_authentication AS vulnerability_cvss2_authentication, vulnerability.cvss2_confidentiality_impact AS vulnerability_cvss2_confidentiality_impact, vulnerability.cvss2_integrity_impact AS vulnerability_cvss2_integrity_impact, vulnerability.cvss2_availability_impact AS vulnerability_cvss2_availability_impact, vulnerability.cvss2_exploitability AS vulnerability_cvss2_exploitability, vulnerability.cvss2_remediation_level AS vulnerability_cvss2_remediation_level, vulnerability.cvss2_report_confidence AS vulnerability_cvss2_report_confidence, vulnerability.cvss2_collateral_damage_potential AS vulnerability_cvss2_collateral_damage_potential, vulnerability.cvss2_target_distribution AS vulnerability_cvss2_target_distribution, vulnerability.cvss2_confidentiality_requirement AS vulnerability_cvss2_confidentiality_requirement, vulnerability.cvss2_integrity_requirement AS vulnerability_cvss2_integrity_requirement, vulnerability.cvss2_availability_requirement AS vulnerability_cvss2_availability_requirement, vulnerability._cvss3_vector_string AS vulnerability__cvss3_vector_string, vulnerability.cvss3_base_score AS vulnerability_cvss3_base_score, vulnerability.cvss3_exploitability_score AS vulnerability_cvss3_exploitability_score, vulnerability.cvss3_impact_score AS vulnerability_cvss3_impact_score, vulnerability.cvss3_base_severity AS vulnerability_cvss3_base_severity, vulnerability.cvss3_temporal_score AS vulnerability_cvss3_temporal_score, vulnerability.cvss3_temporal_severity AS vulnerability_cvss3_temporal_severity, vulnerability.cvss3_environmental_score AS vulnerability_cvss3_environmental_score, vulnerability.cvss3_environmental_severity AS vulnerability_cvss3_environmental_severity, vulnerability.cvss3_attack_vector AS vulnerability_cvss3_attack_vector, vulnerability.cvss3_attack_complexity AS vulnerability_cvss3_attack_complexity, vulnerability.cvss3_privileges_required AS vulnerability_cvss3_privileges_required, vulnerability.cvss3_user_interaction AS vulnerability_cvss3_user_interaction, vulnerability.cvss3_confidentiality_impact AS vulnerability_cvss3_confidentiality_impact, vulnerability.cvss3_integrity_impact AS vulnerability_cvss3_integrity_impact, vulnerability.cvss3_availability_impact AS vulnerability_cvss3_availability_impact, vulnerability.cvss3_exploit_code_maturity AS vulnerability_cvss3_exploit_code_maturity, vulnerability.cvss3_remediation_level AS vulnerability_cvss3_remediation_level, vulnerability.cvss3_report_confidence AS vulnerability_cvss3_report_confidence, vulnerability.cvss3_confidentiality_requirement AS vulnerability_cvss3_confidentiality_requirement, vulnerability.cvss3_integrity_requirement AS vulnerability_cvss3_integrity_requirement, vulnerability.cvss3_availability_requirement AS vulnerability_cvss3_availability_requirement, vulnerability.cvss3_modified_attack_vector AS vulnerability_cvss3_modified_attack_vector, vulnerability.cvss3_modified_attack_complexity AS vulnerability_cvss3_modified_attack_complexity, vulnerability.cvss3_modified_privileges_required AS vulnerability_cvss3_modified_privileges_required, vulnerability.cvss3_modified_user_interaction AS vulnerability_cvss3_modified_user_interaction, vulnerability.cvss3_modified_scope AS vulnerability_cvss3_modified_scope, vulnerability.cvss3_modified_confidentiality_impact AS vulnerability_cvss3_modified_confidentiality_impact, vulnerability.cvss3_modified_integrity_impact AS vulnerability_cvss3_modified_integrity_impact, vulnerability.cvss3_modified_availability_impact AS vulnerability_cvss3_modified_availability_impact, vulnerability.cvss3_scope AS vulnerability_cvss3_scope, vulnerability._cvss4_vector_string AS vulnerability__cvss4_vector_string, vulnerability.cvss4_base_score AS vulnerability_cvss4_base_score, vulnerability.cvss4_base_severity AS vulnerability_cvss4_base_severity, vulnerability.cvss4_attack_vector AS vulnerability_cvss4_attack_vector, vulnerability.cvss4_attack_complexity AS vulnerability_cvss4_attack_complexity, vulnerability.cvss4_attack_requirements AS vulnerability_cvss4_attack_requirements, vulnerability.cvss4_privileges_required AS vulnerability_cvss4_privileges_required, vulnerability.cvss4_user_interaction AS vulnerability_cvss4_user_interaction, vulnerability.cvss4_vulnerable_system_confidentiality_impact AS vulnerability_cvss4_vulnerable_system_confidentiality_imp_1, vulnerability.cvss4_subsequent_system_confidentiality_impact AS vulnerability_cvss4_subsequent_system_confidentiality_imp_2, vulnerability.cvss4_vulnerable_system_integrity_impact AS vulnerability_cvss4_vulnerable_system_integrity_impact, vulnerability.cvss4_subsequent_system_integrity_impact AS vulnerability_cvss4_subsequent_system_integrity_impact, vulnerability.cvss4_vulnerable_system_availability_impact AS vulnerability_cvss4_vulnerable_system_availability_impact, vulnerability.cvss4_subsequent_system_availability_impact AS vulnerability_cvss4_subsequent_system_availability_impact, vulnerability.cvss4_safety AS vulnerability_cvss4_safety, vulnerability.cvss4_automatable AS vulnerability_cvss4_automatable, vulnerability.cvss4_recovery AS vulnerability_cvss4_recovery, vulnerability.cvss4_value_density AS vulnerability_cvss4_value_density, vulnerability.cvss4_vulnerability_response_effort AS vulnerability_cvss4_vulnerability_response_effort, vulnerability.cvss4_provider_urgency AS vulnerability_cvss4_provider_urgency, vulnerability.cvss4_modified_attack_vector AS vulnerability_cvss4_modified_attack_vector, vulnerability.cvss4_modified_attack_complexity AS vulnerability_cvss4_modified_attack_complexity, vulnerability.cvss4_modified_attack_requirements AS vulnerability_cvss4_modified_attack_requirements, vulnerability.cvss4_modified_privileges_required AS vulnerability_cvss4_modified_privileges_required, vulnerability.cvss4_modified_user_interaction AS vulnerability_cvss4_modified_user_interaction, vulnerability.cvss4_modified_vulnerable_system_confidentiality_impact AS vulnerability_cvss4_modified_vulnerable_system_confidenti_3, vulnerability.cvss4_modified_subsequent_system_confidentiality_impact AS vulnerability_cvss4_modified_subsequent_system_confidenti_4, vulnerability.cvss4_modified_vulnerable_system_integrity_impact AS vulnerability_cvss4_modified_vulnerable_system_integrity__5, vulnerability.cvss4_modified_subsequent_system_integrity_impact AS vulnerability_cvss4_modified_subsequent_system_integrity__6, vulnerability.cvss4_modified_vulnerable_system_availability_impact AS vulnerability_cvss4_modified_vulnerable_system_availabili_7, vulnerability.cvss4_modified_subsequent_system_availability_impact AS vulnerability_cvss4_modified_subsequent_system_availabili_8, vulnerability.cvss4_confidentiality_requirement AS vulnerability_cvss4_confidentiality_requirement, vulnerability.cvss4_integrity_requirement AS vulnerability_cvss4_integrity_requirement, vulnerability.cvss4_availability_requirement AS vulnerability_cvss4_availability_requirement, vulnerability.cvss4_exploit_maturity AS vulnerability_cvss4_exploit_maturity, vulnerability.host_id AS vulnerability_host_id, vulnerability.creator_id AS vulnerability_creator_id, vulnerability.update_user_id AS vulnerability_update_user_id, vulnerability.service_id AS vulnerability_service_id 
FROM vulnerability JOIN workspace ON workspace.id = vulnerability.workspace_id 
WHERE vulnerability.workspace_id IN (%(workspace_id_1)s, %(workspace_id_2)s, %(workspace_id_3)s, %(workspace_id_4)s, %(workspace_id_5)s, %(workspace_id_6)s, %(workspace_id_7)s) AND workspace.id = %(id_1)s AND vulnerability.id = %(id_2)s]
[parameters: {'workspace_id_1': (11,), 'workspace_id_2': (10,), 'workspace_id_3': (13,), 'workspace_id_4': (1,), 'workspace_id_5': (3,), 'workspace_id_6': (9,), 'workspace_id_7': (7,), 'id_1': 3, 'id_2': '1755107368857'}]
(Background on this error at: http://sqlalche.me/e/13/9h9h)
2025-08-14T10:28:19+0000 - faraday.server.app - WARNING {Dummy-2255} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.00
2025-08-14T11:10:04+0000 - faraday.server.app - WARNING {Dummy-2855} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (19) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.43
2025-08-14T11:42:03+0000 - faraday.server.app - WARNING {Dummy-3670} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (20) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-08-14T11:45:07+0000 - faraday.server.app - WARNING {Dummy-3670} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (17) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.00
2025-08-14T12:07:07+0000 - faraday.server.app - WARNING {Dummy-3917} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (21) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
2025-08-14T12:07:18+0000 - faraday.server.app - WARNING {Dummy-3917} [pid:23] [app.py:299 - log_queries_count()]  Too many queries done (21) in endpoint workspace_api.WorkspaceView:filter. Maximum query time: 0.01
